modules/apps/dynamic-data-mapping/dynamic-data-mapping-lang/source-formatter.properties

```diff
@@ -0,0 +1 @@
+source.check.PropertiesLanguageKeysCheck.allowedLanguageKeys=a-google-places-api-key-is-required-to-use-this-field
```

Comment:
@rodrigopaulino, can you please give me more details why this is needed?

Thanks

Reply:
Sure, @natocesarrego. Source Formatter actually checks if the messages have markup in them. This line avoids that check, since we need to pass a custom message to our new field and, according to the mockups, some parts should be bold.

Hope this clarifies.

Reply:
Great! Thanks.